covid-policy-pybay-2023 (#3)
BAPyA/References: CovidPolicy.md

# PyBay Attendance Requirements

## Policy Overview

###Vaccine or Negative COVID-19 Test Expectation

1.     [**Outdoor Event Guidelines**](#outdoor-event-guidelines)
1.     [**Vaccine Expectation**](#vaccine-expectation)
1.     [**Negative COVID-19 Test Expectation**](#negative-covid-19-test-expectation)

### Outdoor Event Guidelines

In line with San Francisco guidelines, we prioritize the well-being of our attendees at this outdoor event. While masks and vaccinations are not currently mandated, we reserve the right to change this at any time to comply with San Francisco guidelines. Refunds will be offered for attendees who are unable to comply with updated guidelines. At this time, personal choices regarding their usage are respected. We encourage everyone to uphold good hygiene practices, maintain social awareness, and consider vaccination for the safety of all. Should you feel unwell or exhibit symptoms, we kindly request staying home. Your health and enjoyment are paramount, and we're committed to fostering a secure environment throughout the event. We will refund your ticket if you are unwell.

> We will have masks available at the conference for anyone who needs them. It is also expected that requests for personal space will be respected by attendees and are covered under the code of conduct. 

Reviewer: Maybe we can break this into two parts?

"It is expected that requests for personal space will be respected by attendees. Respecting such requests is covered by the PyBay code of conduct."

Author: agreed.

> While we follow San Francisco's guidance for this outdoor events and don't require masks or proof of vaccination, we respect individual choices. Please prioritize hygiene, personal space, and your health. Stay home if unwell. Your safety matters most as we enjoy the event together.

Reviewer: Either "guidance for outdoor events" or "guidance for this outdoor event"...

Author: agreed.

**At a minimum, we will be following all mandated venue, municipality, and province guidelines. This list, and all protocols, are subject to change if the COVID-19 situation changes.**

> ### Vaccine Expectation

Reviewer: I'm not comfortable with the wording "if you are asked". Can we either commit to ask everyone for proof of vaccination, or explicitly note that vaccination will not be checked, but highly recommended?

Author: That wording is because SF may change it's covid policy to require it, and at that time we would check it.

> If you are asked to show proof of vaccination to attend in person, you will need to show that you are ‘fully vaccinated’ against the COVID-19 virus. We expect that you:

Reviewer 2: ```suggestion
Should local regulations change to require showing vaccination, we expect that you:
```

Author: Changed

*     have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
*     that your most recent dose was at least fourteen days before the conference start

> *     that your most recent dose or booster was no more than twelve months before the conference start

Reviewer 2: ```suggestion
```

Did local regulations ever include this? If not, I suggest removing it.

Author: changed

The following are acceptable forms of proof of full vaccination:

*     Government-issued or government-compliant digital vaccination verification app (such as Clear or VaxYes)
*     Physical (or digital photocopy of) COVID-19 vaccination card, record, or certificate issued by a government agency, public health agency, or other authorized vaccine provider (a photo will also suffice).

The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).

> ### Negative COVID-19 Test Expectation

Reviewer: If we are not requiring vaccination, having something in lieu of that seems also weird. It would be nice if we could commit to, and mention, that we will have some testing kits for people to self-administer before they enter the venue on a voluntary basis.

> In lieu of proof of vaccination, participants may instead show proof of a negative COVID-19 test administered by a verified provider to enter the event.
> 
> *     Provide a negative COVID-19 test upon entry to the event for the first time.
> *     The test must be a PCR, NAAT, or Antigen test taken within 24 hours of picking up your badge on-site.
> *     The test may NOT be an unproctored at-home test. Tests must be administered by a verified provider, either locally in Vancouver or virtually before you travel from your home city (if time permits).
> *     The test results must include your name and the date and time of the test.

Reviewer 2: ```suggestion
We recommend taking a COVID test before attending PyBay. In the event of a positive test result, we offer attendees a refund and ask that they stay home. Additionally, we will have a small supply of tests on hand, should attendees prefer to test on-site.
```

Author: changed
